data-protocols/dsp/dsp-spi/src/main/java/org/eclipse/edc/protocol/dsp/spi/type/DspNamespace.java

```diff
@@ -0,0 +1,51 @@
+/*
+ *  Copyright (c) 2024 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.protocol.dsp.spi.type;
+
+import static org.eclipse.edc.jsonld.spi.Namespaces.DSPACE_SCHEMA;
+
+/**
+ * Enum representing different versions of the DSP (Data Space Protocol) namespace.
+ * Each version is associated with a specific namespace URI.
+ */
+public enum DspNamespace {
```

Comment:
I would model this as a `record`, and then keep the 2 instances as public constants. 
an `enum` is often a blocker for extensibility. I'm not sure we need extensibility here, but I can imagine in the future having a dataspace that wants to use an older version of the protocol while we'll have already deprecated it